samples/webhook-configure-session-parameters.js

```diff
@@ -19,30 +19,13 @@
 // [START dialogflow_cx_v3_webhook_configure_session_parameters]
 
 exports.configureSessionParams = (request, response) => {
-  const tag = request.body.fulfillmentInfo.tag;
-  let newSessionParameter;
-  const text = `${newSessionParameter}. I'm a session parameter configured by the webhook. The webhook's tag is ${tag}.`;
-
-  if (tag === 'month') {
-    newSessionParameter = 'January';
-  } else if (tag === 'year') {
-    newSessionParameter = '1999';
-  }
+  // Session parameter configured by the webhook
```

Comment:
You can't add a comment at any unchanged line of code in this tool, can you?

For line 16, change the description to "Configures a webhook to set a session parameter"